.circleci/config.yml

```diff
@@ -1,9 +1,75 @@
+          shell: powershell.exe
+          command: |
+            # Remove the old go installation
+            Remove-Item "C:\Go" -Recurse
+            # Download golang, unpack it, and then update the PATH to include gobin
+            $golangURI = "https://golang.org/dl/go1.14.7.windows-amd64.zip"
+            $output = "go1.14.7.zip"
+            # The SilentlyContinue is needed to handle access denied error. See
+            # https://discuss.circleci.com/t/access-denied-error-while-trying-to-download-software-on-windows-cirlcleci-environment/32809/2
+            $ProgressPreference = "SilentlyContinue"
+            Invoke-WebRequest -Uri $golangURI -OutFile $output
+            Expand-Archive -LiteralPath $output -DestinationPath "C:\Gotmp"
+            Move-Item "C:\Gotmp\go" "C:\Go"
+            # Verify installation
+            go version
+      - run:
+          name: Install Terraform
+          shell: powershell.exe
+          command: |
+            $TerraformInstallPath = "C:\Program Files\Terraform\terraform.exe"
+            $TerraformTmpPath = "C:\Terraformtmp"
+            $TerraformTmpBinaryPath = "C:\Terraformtmp\terraform.exe"
+            $TerraformPath = "C:\Program Files\Terraform"
+            # Remove any old terraform installation, if present
+            if (Test-Path $TerraformInstallPath)
+            {
+              Remove-Item $TerraformInstallPath -Recurse
+            }
+            # Download terraform and unpack it
+            $terraformURI = "https://releases.hashicorp.com/terraform/1.0.4/terraform_1.0.4_windows_amd64.zip"
+            $output = "terraform.1.0.4.zip"
+            $ProgressPreference = "SilentlyContinue"
+            Invoke-WebRequest -Uri $terraformURI -OutFile $output
+            New-Item -ItemType "directory" -Path $TerraformTmpPath
+            # Unpack Terraform to temp directory
+            Expand-Archive -LiteralPath $output -DestinationPath $TerraformTmpPath
+            # Make new Terraform directory to hold binary
+            New-Item -ItemType "directory" -Path $TerraformPath
+            Move-Item $TerraformTmpBinaryPath $TerraformInstallPath
+            # Add new Terraform path to system
+            $OldPath = [System.Environment]::GetEnvironmentVariable('PATH', "Machine")
+            $NewPath = "$OldPath;$TerraformPath"
+            [Environment]::SetEnvironmentVariable("PATH", "$NewPath", "Machine")
+            # Load System and User PATHs into latest $env:Path, which has the effect of "refreshing" the latest path 
+            # in the current PowerShell session
+            $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User") 
+            # Verify installation
+            terraform version
+      - run:
+          name: Run go tests
+          shell: powershell.exe
+          no_output_timeout: 45m
+          command: |
+            go test -v ./... -timeout 45m
```

Comment:
Are our tests currently passing on Windows?

Reply:
Many are :) and some are not - looks like _some_ failures are possibly due to legitimate Windows quirks we haven't yet handled in Terratest. In Yori's [original POC PR ](https://github.com/gruntwork-io/boilerplate/pull/63/files)for Windows tests he had to handle things like manually removing added carriage returns, etc.

Reply:
Roger :+1: I did get my initial test case passing [here](https://app.circleci.com/pipelines/github/gruntwork-io/terragrunt/961/workflows/6d599171-9ddf-4ccc-956a-3479904aa720/jobs/7245), but there will be some forthcoming changes to attempt to repro the customer-reported issue.